README.md

```diff
@@ -285,6 +303,12 @@ These variables are the defaults of our roles, if you want to override the prope
 | refresh_token_secret | The secret used for refresh tokens jwt signature | Generated by playbook |
 | web_admin_username | Username of the admin user in web application | admin |
 | enable_alerting | Enable the alerting mechanism on web project | false |
+| alert_sender     | Email address used as the "from" address in alerts | alerts@trento-project.io  |
```

Comment:
could we format this like the other rows? only letting one space after the values